please.sh

```diff
@@ -4820,6 +4821,9 @@ make_installers_from_mingw_w64_git () { # [--pkg=<package>[,<package>...]] [--ve
 		output_path="$(cygpath -am "${1#-?}")"
 		output="--output=$output_path" || exit
 		;;
+	--extra-args=*)
```

Comment:
How about calling this `--include-arm64-artifacts=*` instead? I do not necessarily want that `make_installers_from_mingw64_w64_git` function to learn to pass arbitrary flags to the `release.sh` scripts. Something like

```sh
--include-arm64-artifacts=*)
    include_arm64_artifacts="$1"
    ;;
[...]
sh -x "${this_script_path%/*}/$mode/release.sh" $output $extra $include_arm64_artifacts $version
```

Reply:
Sure! Thought it'd be convenient to have a more generic function, but I get that it might add complexity/potential for issues. Done 👍🏼